### PR TITLE
feat(list.txt): add 3 more domains

### PR DESCRIPTION
**Pull request by ali-master**
I've discovered three additional websites that offer temporary email services.

Domains: 
- tempmail.ninja
- etempmail.com
- tempmailto.org

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -55792,3 +55792,6 @@ zzv2bfja5.pl
 zzz.com
 zzzmail.pl
 zzzzzzzzzzzzz.com
+promail.edu.pl
```
[the remark this answers is not in the document]
Comment (ali-master) [reply list.txt]: Done

**Comment by lyrixx** [comment]
The file now conflicts. And the diff is huge. 
Usually diff are really easy to read. Can you make it simpler?
See this PR as a reference: https://github.com/FGRibreau/mailchecker/pull/494

**Comment by ali-master** [comment]
> The file now conflicts. And the diff is huge. Usually diff are really easy to read. Can you make it simpler? See this PR as a reference: #494 And could you squash your commits too?
> 
> Thanks

I updated the PR and checked it out.

**Review by lyrixx — APPROVED** [review]
Thanks

**Pull request merged**